### ISSUE TITLE
[Linux/Windows] Firefox Google Drive Authentication fails

### ISSUE THREAD

**Issue by BestSpaghetti-EUW** [buttercup/buttercup-desktop#1185]
Hello Guys,

I use Linux/Windows as OS (the latter for work) and my browser of choice is Firefox.
Details:

Firefox 109.0 64-Bit
Windows 11 22H2
Manjaro Linux with GNOME DE

When I try to connect my Google Drive to Buttercup, a new Firefox tab opens and I get the normal access control requests for my google account, as shown below:

![access_rights](https://user-images.githubusercontent.com/78494236/214130203-a34687eb-bbc1-400a-9fa9-a2d648dbd75c.png)

So far so good. Then when I click on "Continue" I get the following message that an error occured:

![error_occured](https://user-images.githubusercontent.com/78494236/214130499-67053a41-dedb-4d89-a01e-febe93359c15.png)

When I try the same procedure with a Chromium-based browser, I don't get the error message but then am stuck on the [buttercup.pw-URL](https://buttercup.pw/). Probably the same bug as in #987

**Comment by perry-mitchell** [comment]
I don't think this is related to #987, but can you kindly translate the error message to English in case I'm missing something?

**Comment by BestSpaghetti-EUW** [comment]
Dammit, the error was produced by a firefox addon. I'm gonna close the error now.
